models/user.js

```diff
@@ -156,7 +156,7 @@ const loginUser = async (data) => {
     if (!passwordIsValid) {
       throw createError(401, "Incorrect username or password");
     }
-
+    delete user.password;
```

Comment:
You probably would have caught this, but we shouldn't send the hashed password back to the front end. The back end should simply let the front end know that the user is authenticated.